paasta_tools/long_running_service_tools.py

```diff
@@ -185,7 +185,7 @@ def get_bounce_priority(self) -> int:
         """
         return self.config_dict.get('bounce_priority', 0) * -1
 
-    def get_instances(self) -> int:
+    def get_instances(self, with_limit=True) -> int:
```

Comment:
Sorry to nitpick but can you add a type too (we're trying to add them whenever we add new stuff)
The syntax is like: `get_instances(self, with_limit: bool=True) -> int:`